cmd/vela-worker/register.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-vela/types/library"
+	"github.com/sirupsen/logrus"
+)
+
+// register is a helper function to register
+// the worker in the database, updating the item
+// if the worker already exists
+func (w *Worker) register(config *library.Worker) error {
+	// check to see if the worker already exists in the database
+	_, resp, err := w.VelaClient.Worker.Get(config.GetHostname())
+	if err != nil {
+		// check to see if the response was nill
```

Comment:
I think this is a typo

```suggestion
		// check to see if the response was nil
```